src/db_impl_gc.cc

```diff
@@ -81,9 +81,10 @@ Status TitanDBImpl::BackgroundGC(LogBuffer* log_buffer) {
         cfh = db_impl_->GetColumnFamilyHandleUnlocked(column_family_id);
         assert(column_family_id == cfh->GetID());
         blob_gc->SetColumnFamily(cfh.get());
-        if (blob_gc->trigger_next()) {
+        if (blob_gc->trigger_next() && gc_queue_.size() < 50) {
```

Comment:
I'm not sure whether to set a value directly or add a field in TitanCFOptions and make it adjustable, and not sure the appropriate value should be here

Reply:
Instead of limiting the queue size, how about we limit the number of jobs in queue for the CF does not exceed max_background_gc?

Reply:
but if the number of jobs is too small(like that we set max_bg_gc = 1), trigger_next may never triggered

Reply:
max_bg_gc+1 then? Anyway it seems to me the only factor that would affect queue size is how many background GC thread we have.